javalib/src/main/scala/java/util/TimerTask.scala

```diff
@@ -0,0 +1,30 @@
+package java.util
+
+import scala.scalajs.js.timers
+
+abstract class TimerTask {
+  private[util] var cancelled: Boolean = false
+  private[util] var lastScheduled: Long = 0L
+
+  def run(): Unit
+
+  def cancel(): Unit = {
+    cancelled = true
+  }
```

Comment:
Indeed, I had not tested this yet, it was meant just as a strawman. It indeed should remove itself from the `tasks` list, or even better, there should be no `tasks` (although it's unclear to me if `TimerTask`s can go to multiple `Timer`s).